internal/pkg/agent/cmd/inspect.go

```diff
@@ -130,20 +130,23 @@ type inspectConfigOpts struct {
 func inspectConfig(ctx context.Context, cfgPath string, opts inspectConfigOpts, streams *cli.IOStreams) error {
 	l, err := newErrorLogger()
 	if err != nil {
-		return err
+		return fmt.Errorf("error creating logger: %w", err)
 	}
 
 	if !opts.variables && !opts.includeMonitoring {
 		fullCfg, err := operations.LoadFullAgentConfig(ctx, l, cfgPath, true)
 		if err != nil {
-			return err
+			return fmt.Errorf("error loading agent config: %w", err) //I bet it's here
```

Comment:
Is this a leftover comment?

Reply:
It's me going mad trying to find https://github.com/elastic/go-ucfg/pull/197, yes